python/ray/ml/train/impl/function_trainer.py

```diff
@@ -0,0 +1,55 @@
+import abc
+from typing import Dict, Any, Optional
+
+from ray.ml.checkpoint import Checkpoint
+from ray.ml.train import Trainer
+from ray.tune import PlacementGroupFactory
+from ray.util.annotations import DeveloperAPI
+
+
+@DeveloperAPI
+class FunctionTrainer(Trainer):
+    """A Trainer that provides a function interface.
+
+    When subclassing ``FunctionTrainer``, you only need to override the
+    ``training_function`` and ``resource_function`` methods. There is no
+    need to directly work with Trainables.
+
+    Args:
+        Same as ray.ml.Trainer.
```

Comment:
nah, let's be explicit here :D

Reply:
+100

Reply:
Got it, yep this is no longer applicable now that `FunctionTrainer` is the base Trainer. All args explicitly documented.